src/main/java/seedu/guest/ui/GuestCard.java

```diff
@@ -54,14 +59,112 @@ public GuestCard(Guest guest, int displayedIndex) {
+        Tooltip emailTT = new Tooltip("Email");
+        email.setTooltip(emailTT);
+
         room.setText(guest.getRoom().value);
+        Image roomImage = new Image(getClass().getResourceAsStream("/images/room.png"));
+        ImageView setRoomImage = new ImageView(roomImage);
+        setRoomImage.setFitWidth(16);
+        setRoomImage.setFitHeight(16);
+        room.setGraphic(setRoomImage);
+        room.setGraphicTextGap(7);
+        room.setWrapText(true);
+        Tooltip roomTT = new Tooltip("Room Number");
+        room.setTooltip(roomTT);
+
         dateRange.setText(guest.getDateRange().value);
-        numberOfGuests.setText("No. of Guests: " + guest.getNumberOfGuests().value);
-        isRoomClean.setText("Room Cleaned: " + guest.getIsRoomClean().value);
-        bill.setText("Bill: " + guest.getBill().value);
-        request.setText("Request: " + guest.getRequest().value);
+        Image dateRangeImage = new Image(getClass().getResourceAsStream("/images/dateRange.png"));
+        ImageView setDateRangeImage = new ImageView(dateRangeImage);
+        setDateRangeImage.setFitWidth(16);
+        setDateRangeImage.setFitHeight(16);
+        dateRange.setGraphic(setDateRangeImage);
+        dateRange.setGraphicTextGap(7);
+        dateRange.setWrapText(true);
+        Tooltip dateRangeTT = new Tooltip("Date of Stay");
+        dateRange.setTooltip(dateRangeTT);
+
+        numberOfGuests.setText(guest.getNumberOfGuests().value);
+        Image numberOfGuestsImage = new Image(getClass().getResourceAsStream("/images/group.png"));
+        ImageView setNumberOfGuestsImage = new ImageView(numberOfGuestsImage);
+        setNumberOfGuestsImage.setFitWidth(16);
+        setNumberOfGuestsImage.setFitHeight(16);
+        numberOfGuests.setGraphic(setNumberOfGuestsImage);
+        numberOfGuests.setGraphicTextGap(7);
+        Tooltip numberOfGuestsTT = new Tooltip("Number of Guests");
+        numberOfGuests.setTooltip(numberOfGuestsTT);
+
+        isRoomClean.setText("");
+        isRoomCleanStatus.setText("");
+        Image isRoomCleanImage = new Image(getClass().getResourceAsStream("/images/isRoomClean.png"));
+        ImageView setIsRoomCleanImage = new ImageView(isRoomCleanImage);
+        setIsRoomCleanImage.setFitWidth(16);
+        setIsRoomCleanImage.setFitHeight(16);
+        isRoomClean.setGraphic(setIsRoomCleanImage);
+
+        // Set graphic based on status of isRoomClean
+        if (guest.getIsRoomClean().value.equals("yes")) {
+            Image isRoomCleanStatusImage = new Image(getClass().getResourceAsStream("/images/roomClean.png"));
+            ImageView setIsRoomCleanStatusImage = new ImageView(isRoomCleanStatusImage);
+            setIsRoomCleanStatusImage.setFitWidth(18);
+            setIsRoomCleanStatusImage.setFitHeight(18);
+            isRoomCleanStatus.setGraphic(setIsRoomCleanStatusImage);
+        } else {
+            Image image = new Image(getClass().getResourceAsStream("/images/roomNotClean.png"));
```

Comment:
Should the name of the image be roomUnclean instead of NotClean to standardise the naming convention?

Reply:
Agreed and changed.